src/main/java/com/google/api/generator/gapic/model/HttpBindings.java

```diff
@@ -36,21 +36,54 @@ public enum HttpVerb {
 
   @AutoValue
   public abstract static class HttpBinding implements Comparable<HttpBinding> {
+
+    // The fully qualified name of the field. e.g. request.complex_object.another_object.name
     public abstract String name();
 
     abstract String lowerCamelName();
 
-    public abstract boolean isOptional();
+    // An object that contains all info of the leaf level field
+    @Nullable
+    public abstract Field field();
 
-    public abstract boolean isRepeated();
+    public boolean isOptional() {
```

Comment:
These code looks like from my another PR, you may need to merge from main again.

Reply:
Good catch - updated!